gammapy/analysis/tests/test_analysis.py

```diff
@@ -262,6 +262,44 @@ def test_analysis_1d_stacked():
     assert_allclose(pars["index"].value, 2.76913, rtol=1e-2)
     assert_allclose(pars["amplitude"].value, 5.479729e-11, rtol=1e-2)
 
+@requires_dependency("iminuit")
+@requires_data()
+def test_analysis_1d_stacked_no_fit_range():
```

Comment:
Maybe one could just remove the fit range definition from one of the existing tests to avoid introducing a new one, that runs a full analysis again? Just to save a bit of runtime for the tests...